src/controllers/ContactController.ts

```diff
@@ -19,10 +19,11 @@ export class ContactController extends Controller {
     const emailOptionalFunctions = [
       ContactFunction.SIGNATORY_AUTHORIZED,
       ContactFunction.ASSISTING,
+      ContactFunction.OLD,
     ];
     await validate([
       body('gender').isIn(Object.values(Gender)),
-      body('firstName').trim(),
+      body('firstName').optional({ values: 'falsy' }).isString().trim(),
```

Comment:
This field is not optional when the email also is not optional, so it should be configured the same way as `email`.